src/components/Fetch/Highlights.js

```diff
@@ -0,0 +1,91 @@
+import Button from 'react-bootstrap/lib/Button';
```

Comment:
This file is a great candidate for enabling flow on. Do all of the other stuff here first, then in a separate commit/branch let's add flow here.